client/src/views/Game.vue

```diff
@@ -73,6 +74,18 @@
               <p>
                 <v-switch v-model="pane_lock" :label="`Unlock Players Pane`"></v-switch>
               </p>
+
+              <!-- Need Help? button -->
+              <v-btn @click.native="helpMenu = !helpMenu" >Need Help?</v-btn>
+              <v-card v-show="helpMenu" class="mt-5 pa-2" outlined tile >                  
+                <router-link to="/help#rules">Rules</router-link> |
```

Comment:
So, it looks like we were never able to resolve the ability to actually hit the #anchor tags. My opinion is that I think this should be consolidated to a single link to the help page for now.

Reply:
I didn't even notice that, haha! Ok, I'll just drop them. The page isn't that big anyway

Reply:
Yeah. I think a single link with a `help` keyword is probably all we need here.

Reply:
@logan-hendricks  
I may have over achieved and I made the anchor tags work instead of deleting them, haha

Reply:
That's also fine if you got them working. I could have sworn we did that change earlier and it didn't work out.
but I'll be happy to test it here.

Reply:
Ah. I looked at implementing the scrollToBehavior method. but the way you have it there makes waaayyy more sense than how I understood it to work. It basically does an override. Good call. that now works.

Reply:
Sweet! Thanks